src/charts/onPreprocess/filterData.js

```diff
@@ -0,0 +1,5 @@
+export default function filterData() {
+    this.raw_data = this.measure_data;
+    if (!this.config.inliers) this.raw_data = this.raw_data.filter(d => d.abnormalID);
```

Comment:
Kind of a weird use case here where someone has an outlier during unscheduled visits, but not other visits. Right now I think that person is shown when both inliers and unscheduled visits are hidden ... That's probably a reasonable design decision, but it might be a bit confusing for the user since there will be a person who *appears* to be an inlier showing when inliers isn't checked.

Reply:
Yeah, good point...  The inliers are defined by the entirety of their results for a single measure.